src/Sylius/Behat/Context/Transform/OrderContext.php

```diff
@@ -47,6 +58,21 @@ public function getOrderByNumber($orderNumber)
     }
 
     /**
+     * @Transform /^this order made by "([^"]+)"$/
+     */
+    public function getOrderByCustomer($email)
+    {
+        $customer = $this->customerRepository->findOneBy(['email' => $email]);
+        Assert::notNull($customer, sprintf('Cannot find customer with email %s.', $email));
```

Comment:
If we use this transformer then why we have https://github.com/Sylius/Sylius/pull/5319/files#r68388816?